src/sentry/receivers/releases.py

```diff
@@ -18,7 +19,30 @@ def resolve_group_resolutions(instance, created, **kwargs):
     clear_expired_resolutions.delay(release_id=instance.id)
 
 
+def remove_resolved_link(link):
+    # TODO(dcramer): ideally this would simply "undo" the link change,
+    # but we don't know for a fact that the resolution was most recently from
+    # the GroupLink
```

Comment:
Would having a 'resolution history' help here? Could it be used for other purposes as well?

Reply:
It might but I think that gets into a sticky situation where you might want other history. Activity could probably be used for this, we'd just have to bucket groups of items into things. e.g. all the SET_RESOLVED_IN_X get bucketed into a resolutions category, so we could just find the last resolution.

Reply:
Activity at least lets us communicate to the user what happened, and give them a trail on how the resolution ended up in the current state. Commit messages are generally immutable so I don't forsee this being a big problem.

Reply:
Ok this is less bad than I thought.

1. my query was wrong
2. adding in last_seen severely truncates results

there are 270 orphaned issues that are still happening and have a resolution associated:

```
GroupLink.objects.filter(
    linked_type=GroupLink.LinkedType.commit,
    relationship=GroupLink.Relationship.resolves,
).extra(
    where=[
        "NOT EXISTS(SELECT 1 FROM sentry_releasecommit where commit_id = sentry_grouplink.linked_id)",
        "EXISTS(SELECT 1 FROM sentry_groupedmessage where id = sentry_grouplink.group_id AND status = 0 AND last_seen > now() - interval '7 days')",
    ]
)
```